src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+  height: 60px;
+}
+
+.nav__list {
+  padding: 0;
+}
+
```

Comment:
you need to remove list stylling form the list by 
`list-style: none;`

Reply:
i tried it but the styling was not removed (it did not affect)